README.md

```diff
@@ -101,3 +107,18 @@ the metrics will rather look like:
 In case one of your target urls contains a `=` character (for instance consul agent's exporter is available at `/v1/agent/metrics?format=prometheus`), you **must** use the custom labelling notation:
 
      bin/prometheus-aggregate-exporter -targets="consul=http://localhost:8500/v1/agent/metrics?format=prometheus"
+
+#### Dynamic registration 
+
+If `targets.dynamic.registration` is set to `true` additional `HTTP GET` endpoints are available `/register` and `/unregister`.
+Those can be used for example for dynamic exporter discovery in multi replica `Kubernetes` environment when exact adress of exporter is not known at start.
+
+Both support query parameters:
+* name - name of registered exporter (same as name usable in `targets`)
+* address - address of exporter without schema (ex. `localhost:9090/metrics`)
+* schema - exporter schema (default `http`)
+
+Example usage:
+* register exporter: `localhost:8080/register?name=someExporter&address=localhost:3000/metrics`
```

Comment:
should we enforce a particular HTTP method e.g. POST or PATCH in these handlers?

Reply:
I would rather not. 
This is more an URL handler rather than proper REST resources, enforcing ex. PUT and DELETE (in my opinion those would be most suited) will require additional configuration at client part (ex. -X PUT for curl) and does not provide actual value.